WcaOnRails/.gitignore

```diff
@@ -39,16 +39,16 @@ pids/*.pid
 # Coveralls stuff
 /coverage/
 
-/public/packs
-/public/packs-test
-/node_modules
```

Comment:
Why are you removing this from the gitignore?
Developers running the website locally with docker-compose will still produce these files and they're not supposed to check them in to VCS

Reply:
honestly no idea how this got in here

Reply:
ah I removed it because it's duplicated 7 lines above it!